src/Symfony/Component/Security/Core/Authorization/Voter/AbstractVoter.php

```diff
@@ -89,11 +88,7 @@ protected function isClassInstanceOf($actualClass, $expectedClass)
     }
 
     /**
-     * Perform a single access check operation on a given attribute, object and (optionally) user
-     * It is safe to assume that $attribute and $object's class pass supportsAttribute/supportsClass
-     * $user can be one of the following:
-     *   a UserInterface object (fully authenticated user)
-     *   a string               (anonymously authenticated user).
+     * Perform a single access check operation on a given attribute, object and token.
```

Comment:
I would still like to add that it's safe to assume that `$attribute` and `$object` passed the `supports` method.